lighthouse-cli/bin.ts

```diff
@@ -83,18 +83,17 @@ const cliFlags = yargs
     'port': 'The port to use for the debugging protocol. Use 0 for a random port',
     'skip-autolaunch': 'Skip autolaunch of Chrome when already running instance is not found',
     'select-chrome': 'Interactively choose version of Chrome to use when multiple installations are found',
+    'interactive': 'Open Lighthouse in interactive mode'
```

Comment:
Guess it's time to add this now. Because there is at least a rerun button to let users interact with LH.